src/components/emailconfiguratin.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import signinlogo from '../assets/signinlogo.svg';
+const Configuration = () => {
```

Comment:
Please rename to `Confirmation` and the file name too should be EmailConfirmation.js